src/pages/List.js

```diff
@@ -31,7 +18,7 @@ const List = () => {
         path="items"
         // Sort the data
         sort="name"
-        filter={['token', '==', 'baron flew gibby']}
+        filter={['token', '==', getStoredToken()]}
```

Comment:
No more hard coded token. It's using the token stored in local storage. I tested this by starting another server on a new port and adding new items to the shopping list.

![List associated with token](https://p112.p2.n0.cdn.getcloudapp.com/items/ApuOzzEG/Image+2020-01-01+at+7.20.43+PM.png?v=06b11e12a800e2bc264e1dd4b1d72089)

Reply:
Nice!